kdbplus/src/ipc/connection.rs

```diff
@@ -553,12 +550,8 @@ impl QStream {
                         .expect("failed to accept TLS connection");
                 }
                 // TLS is always a remote connection
-                let mut qstream = QStream::new(
-                    Box::new(TlsStream::from(tls_socket)),
-                    ConnectionMethod::TCP,
-                    true,
-                    false,
-                );
+                let mut qstream =
+                    QStream::new(Box::new(tls_socket), ConnectionMethod::TCP, true, false);
```

Comment:
Can we omit `TlsStream::from(` ?

Reply:
I think so, unless you need to copy it, in which case tls_socket.clone() is probably better

I don't have a q subscription so I can't test fully but this does compile

Reply:
Yes, the type of `tls_socket` is `TlsStream<TcpStream>`